io.elementary.Sdk.json

```
@@ -19,8 +19,8 @@
     "finish-args": [
         "--env=GI_TYPELIB_PATH=/app/lib/girepository-1.0",
         "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/share",
-        "--sdk=io.elementary.Sdk//@SDK_BRANCH@",
-        "--runtime=io.elementary.Platform//@SDK_BRANCH@"
+        "--sdk=io.elementary.Sdk//3.38",
+        "--runtime=io.elementary.Platform//3.38"
```

I believe theses are to be `daily`
```suggestion
        "--sdk=io.elementary.Sdk//daily",
        "--runtime=io.elementary.Platform//daily"
```

---

Yep. I'll update it once this build finishes and the cache gets set